AutomatedTesting/Gem/PythonTests/prefab/Prefab_Operations.py

```diff
@@ -0,0 +1,190 @@
+    file_path = prefab_test_utils.get_prefab_file_path(file_name)
+    create_prefab_result = prefab.PrefabPublicRequestBus(bus.Broadcast, 'CreatePrefabInMemory', entity_ids, file_path)
+    Report.result(OperationResults.create_entity_succeeded, create_prefab_result.IsSuccess())
+    
+    if not create_prefab_result.IsSuccess():
+        return EntityId()
+
+    container_entity_id = create_prefab_result.GetValue()
+
+    if prefab_name is not None:
+        editor.EditorEntityAPIBus(bus.Event, 'SetName', container_entity_id, prefab_name)
+    else:
+        prefab_name = file_name
+
+    prefab_test_utils.wait_for_propagation()
+    container_entity_id = prefab_test_utils.find_entity_by_unique_name(prefab_name)
+    return container_entity_id
+
+
+def instantiate_prefab(file_name, prefab_name=None, parent_entity_id=EntityId(), expected_prefab_position=Vector3()):
+    file_path = prefab_test_utils.get_prefab_file_path(file_name)
+    instantiate_prefab_result = prefab.PrefabPublicRequestBus(
+        bus.Broadcast, 'InstantiatePrefab', file_path, parent_entity_id, expected_prefab_position)
+    Report.result(OperationResults.instantiate_prefab_succeeded, instantiate_prefab_result.IsSuccess())
+
+    if not instantiate_prefab_result.IsSuccess():
+        return EntityId()
+
+    container_entity_id = instantiate_prefab_result.GetValue()
+
+    if prefab_name is not None:
+        editor.EditorEntityAPIBus(bus.Event, 'SetName', container_entity_id, prefab_name)
+    else:
+        prefab_name = file_name
+
+    prefab_test_utils.wait_for_propagation()
+    container_entity_id = prefab_test_utils.find_entity_by_unique_name(prefab_name)
+    prefab_test_utils.check_entity_at_position(container_entity_id, expected_prefab_position)
+
+    return container_entity_id
+
+
+def delete_prefabs(container_entity_ids):
+    instances_to_remove_name_counts = Counter()
+    instances_removed_expected_name_counts = Counter()
+
+    entity_ids_to_remove = list(container_entity_ids)
+    while entity_ids_to_remove:
+        entity_id = entity_ids_to_remove.pop(-1)
+        entity_name = editor.EditorEntityInfoRequestBus(bus.Event, 'GetName', entity_id)
+        instances_to_remove_name_counts[entity_name] += 1
+
+        children_entity_ids = editor.EditorEntityInfoRequestBus(bus.Event, 'GetChildren', entity_id)
+        for child_entity_id in children_entity_ids:
+            entity_ids_to_remove.append(child_entity_id)
+
+    for entity_name, entity_count in instances_to_remove_name_counts.items():
+        entities = prefab_test_utils.find_entities_by_name(entity_name)
+        instances_removed_expected_name_counts[entity_name] = len(entities) - entity_count
+
+    delete_prefab_result = prefab.PrefabPublicRequestBus(bus.Broadcast, 'DeleteEntitiesAndAllDescendantsInInstance', container_entity_ids)
+    Report.result(OperationResults.delete_prefab_succeeded, delete_prefab_result.IsSuccess())
+
+    if not delete_prefab_result.IsSuccess():
+        return
```

Comment:
this breaks the flow of the result reporting. If that is the case I recommend to use Report.critical_result in line 136

Reply:
Will fix it.

Reply:
Fixed.